generators/server/templates/app/src/test/java/TestApplication.java

```diff
@@ -13,6 +13,11 @@
 <%_ if (databaseType === 'mariadb') { _%>
 import org.testcontainers.containers.MariaDBContainer;
 <%_ } _%>
+<%_ if (features.includes('localstack')) { _%>
+import org.springframework.test.context.DynamicPropertyRegistry;
+import org.testcontainers.containers.localstack.LocalStackContainer;
+<%_ } _%>
+import org.testcontainers.utility.DockerImageName;
     
 @TestConfiguration(proxyBeanMethods = false)
 public class TestApplication {
```

Comment:
Usually a common approach is having a separate `ContainersConfig.java` and `TestApplication.java`

Reply:
I got inspire from start.spring.io generated code where both are present in same class. earlier, I used to manage two classes as you have suggested. As I don't see any significant gain creating two classes, can we continue in a single class?

Reply:
I still see start.spring.io generating 2 different files: https://start.spring.io/#!type=maven-project&language=java&packaging=jar&jvmVersion=17&groupId=com.sivalabs&artifactId=myapp&name=myapp&description=Demo project for Spring Boot&packageName=com.sivalabs.myapp&dependencies=lombok,web,data-jpa,flyway,h2,postgresql,validation,actuator,devtools,configuration-processor,testcontainers

Reply:
Aah, my bad. Yes it's generating a single file.